sdk/storage/Azure.Storage.Queues/tests/QueueTestBase.cs

```diff
@@ -136,27 +136,40 @@ public StorageSharedKeyCredential GetNewSharedKeyCredentials()
                 TestConfigDefault.AccountKey);
 
         public SasQueryParameters GetNewAccountSasCredentials(StorageSharedKeyCredential sharedKeyCredentials = default)
-            => new AccountSasBuilder
+        {
+            var builder = new AccountSasBuilder
             {
                 Protocol = SasProtocol.None,
                 Services = AccountSasServices.Queues,
                 ResourceTypes = AccountSasResourceTypes.Container,
                 StartsOn = Recording.UtcNow.AddHours(-1),
                 ExpiresOn = Recording.UtcNow.AddHours(+1),
-                Permissions = new QueueAccountSasPermissions { Read = true, Write = true, Update = true, Process = true, Add = true, Delete = true, List = true }.ToString(),
                 IPRange = new SasIPRange(IPAddress.None, IPAddress.None)
-            }.ToSasQueryParameters(sharedKeyCredentials);
+            };
+            builder.SetPermissions(
+                AccountSasPermissions.Read |
```

Comment:
QueueAccountSasPermissions

Reply:
It is using an AccountSasBuilder though, so I think it was incorrect before.